src/Pickles/Pickles.TestFrameworks.UnitTests/MsTest/WhenParsingMsTestResultsFileWithIgnoredExample.cs

```diff
@@ -0,0 +1,65 @@
+﻿//  --------------------------------------------------------------------------------------------------------------------
+//  <copyright file="WhenParsingMsTestResultsFileWithEmptyExampleValues.cs" company="PicklesDoc">
+//  Copyright 2011 Jeffrey Cameron
+//  Copyright 2012-present PicklesDoc team and community contributors
+//
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//  </copyright>
+//  --------------------------------------------------------------------------------------------------------------------
+
+
+using NUnit.Framework;
+using PicklesDoc.Pickles.ObjectModel;
+using PicklesDoc.Pickles.TestFrameworks.MsTest;
+using System.Collections.Generic;
+using NFluent;
+
+namespace PicklesDoc.Pickles.TestFrameworks.UnitTests.MsTest
+{
+    [TestFixture]
+    public class WhenParsingMsTestResultsFileWithIgnoredExample : StandardTestSuite<MsTestResults>
+    {
+        public WhenParsingMsTestResultsFileWithIgnoredExample()
+            : base("MsTest." + "results-example-mstest-ignoredexample.trx")
```

Comment:
sorry. kinda lost you here. But I'll try to answer :)
In the new code, the search on the row 2 (ignored) will still fail, but GetExampleResult method will handle it gracefully (check on null) and treat it as Inconclusive so that the whole cycle can be completed